web/massastation/src/pages/Index/Index.tsx

```diff
@@ -34,23 +34,21 @@ export function Index() {
   const [refreshPlugins, setRefreshPlugins] = useState(0);
   const theme = useConfigStore((s) => s.theme);
 
-  const { data: plugins } = useResource<PluginHomePage[]>('plugin-manager');
+  const { data: plugins } = useResource<IMassaPlugin[]>('plugin-manager');
 
-  const { data: availablePlugins } =
-    useResource<PluginStoreItemRequest[]>('plugin-store');
+  const { data: availablePlugins } = useResource<IMassaStore[]>('plugin-store');
 
   const { mutate, isSuccess, isError, isLoading } =
     usePost<null>('plugin-manager');
-  const walletName = 'Massa Wallet';
 
   useEffect(() => {
     const isWalletInstalled = plugins?.some(
-      (plugin: PluginHomePage) => plugin.name === walletName,
+      (plugin: IMassaPlugin) => plugin.name === 'wallet',
```

Comment:
"wallet"? Are you sure?

Reply:
no ... I forgot to push, should be good now